### PR TITLE
fix: mask sensitive data on sentry

### PR DESCRIPTION
**Pull request by ddwsd**
Mask sensitive data on sentry!

### CODE REVIEW

File: src/hooks/usePastTransaction/usePastTransaction.tsx [hunk 1]
```diff
@@ -33,7 +33,11 @@ export const usePastTransaction = (
         );
         setPastTransactionsResult(pastTransactionsResponse?.pastTransactions);
       } catch (error) {
-        Sentry.captureException(`Unable to fetch past transactions: ${ids}`);
+        Sentry.captureException(
+          `Unable to fetch past transactions: ${ids.map(id =>
+            id.replace(/\w/g, "*")
```
Comment (szenius) [inline src/hooks/usePastTransaction/usePastTransaction.tsx:38]: Do we want to leave the last 4 chars for debugging?

Comment (ddwsd) [reply src/hooks/usePastTransaction/usePastTransaction.tsx:38]: for nrics it doesn't make a difference i think, but yeah for the other types of identification

Comment (szenius) [reply src/hooks/usePastTransaction/usePastTransaction.tsx]: What do you mean it doesn't make a difference ah

Comment (szenius) [reply src/hooks/usePastTransaction/usePastTransaction.tsx]: Btw in the backend we use `string.padStart()` to do masking, maybe we want to be consistent with that~ Can check out https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart

Comment (ddwsd) [reply src/hooks/usePastTransaction/usePastTransaction.tsx]: nice, thanks! was wondering what's the best way lol. 
i think the nrics would be masked in our backend and db anyway, so it's a bit harder to debug

**Review by szenius — APPROVED** [review]
bitmoji

**Pull request merged**